src/extension-consumer/Cargo.toml

```diff
@@ -36,9 +36,9 @@ which = "4.0.2"
 # Fluvio dependencies
 
 flv-util = { version = "0.5.0" }
-fluvio-future = { version = "0.1.8" }
+fluvio-future = { version = "0.1.8", features = ["fs", "io"] }
```

Comment:
Not sure why this broke but when I pulled master, I had to add this to fix the build.

Reply:
shouldn't need that. what is exact error?

Reply:
I notice that I don't get the error when I just build the fluvio cli but when I `cargo build` in the `extension-common` subcrate, I get this:

```
simlay@tower-1:~/projects/infinyon/fluvio/src/extension-consumer (extension-command-protocol)$ cargo build
   Compiling fluvio-extension-common v0.1.0 (/home/simlay/projects/infinyon/fluvio/src/extension-common)
   Compiling fluvio-extension-consumer v0.1.0 (/home/simlay/projects/infinyon/fluvio/src/extension-consumer)
error[E0432]: unresolved import `fluvio_future::fs`
   --> src/extension-consumer/src/produce/mod.rs:118:24
    |
118 |     use fluvio_future::fs::File;
    |                        ^^ could not find `fs` in `fluvio_future`

error[E0432]: unresolved import `fluvio_future::io`
   --> src/extension-consumer/src/produce/mod.rs:119:24
    |
119 |     use fluvio_future::io::stdin;
    |                        ^^ could not find `io` in `fluvio_future`

error[E0432]: unresolved import `fluvio_future::io`
   --> src/extension-consumer/src/produce/mod.rs:120:24
    |
120 |     use fluvio_future::io::ReadExt;
    |                        ^^ could not find `io` in `fluvio_future`

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0432`.
error: could not compile `fluvio-extension-consumer`

To learn more, run the command again with --verbose.
```

Reply:
Yup.  consumer crates need: `io` and `fs` feature flags